### ISSUE TITLE
Add support for publishing to Mavel local

### ISSUE THREAD

**Issue by tlinkowski** [tlinkowski/tlinkowski-superpom#8]

**Comment by tlinkowski** [comment]
Fixed with #5 by applying Kordamp's `publishing` plugin (part of `project` plugin).

**Comment by tlinkowski** [comment]
To use a locally published plugin, add to `settings.gradle`:
```kotlin
pluginManagement {
  repositories {
    mavenLocal {
      content {
        includeGroup("pl.tlinkowski.tlinkowski-superpom")
      }
    }
    gradlePluginPortal()
  }
}
```
